Database/Corrections/cataQuestFixes.lua

```diff
@@ -3675,6 +3675,9 @@ function CataQuestFixes.Load()
         [26116] = { -- Kinelory Strikes
             [questKeys.triggerEnd] = {"Protect Kinelory", {[zoneIDs.ARATHI_HIGHLANDS] = {{54.85,55.55}}}},
         },
+        [26118] = { -- Seize the Ambassador
+            [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
```

Comment:
This is correct. However an `extraObjective` pointing to "Ambassador Slaghammer" would be even better. So not only the target position is shown by Questie.

Reply:
47632